tasks/facial_landmark_detection/model.py

```diff
@@ -0,0 +1,169 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import torch
+from typing import Callable, List, Optional
+
+from torch import nn, Tensor
+from torchvision.models._utils import _make_divisible
+from torchvision.ops.misc import Conv2dNormActivation
+
+class CustomInvertedResidual(nn.Module):
```

Comment:
Where did this stuff come from? Can you point to a paragraph in the paper? Does it exist in some other git repo?  Did you invent it? Love to know the thinking behind this...

Reply:
I wished I invented it! :)

Reply:
thanks